en-US/O.xml

```diff
@@ -108,7 +108,29 @@
 			</listitem>
 
 		</varlistentry>
-<!-- Add entries for "on-prem", "on-premise", "on-premises"? -->
+
+<!-- Advice differs from IBM Style, which says to use "on premises", "on-premises". -->
+		 <varlistentry id="on premises">
+			<term>on premise, on-premise</term>
+			 <listitem>
+				<para>
+					<emphasis>adjective, adverb</emphasis> Write as shown. Use "on-premise" as an adjective before a noun. Otherwise, use "on premise." Do not use "on premises", "on-premises."
+				</para>
+
+			</listitem>
+
+		</varlistentry>
+
+		 <varlistentry id="on premises">
```

Comment:
Same comments as above.
I'm also trying to follow the "one sentence per line" rule in this guide as well.
Note, too, that "Do not use" should be a sentence on its own.